package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "spruce",
-  "version": "0.5.1",
+  "version": "0.5.2",
```

Comment:
Deploying to production this morning bumped the version in my package.json. Do ya'll think I should do this in a separate PR?